test-configs.yaml

```diff
@@ -461,6 +461,24 @@ device_types:
     filters:
       - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
 
+  meson_g12a_sei510:
+    name: 'meson-g12a-sei510'
+    mach: amlogic
+    class: arm64-dtb
+    boot_method: uboot
+    flags: ['little_endian']
+    filters:
+      - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
+
+  meson-g12a-x96-max:
+    name: 'meson-g12a-x96-max'
```

Comment:
The `name` property is not necessary when it's the same as the config name, so that can be dropped as well.